hummingbird/ml/convert.py

```diff
@@ -56,13 +83,15 @@ def _convert_sklearn(model, test_input=None, extra_config={}):
     assert model is not None
     assert torch_installed(), "To use Hummingbird you need to install torch."
 
+    from .ir_converters.topology import convert as topology_converter
+
     # Parse scikit-learn model as our internal data structure (i.e., Topology)
-    # We modify the scikit learn model during optimizations.
+    # We modify the scikit learn model during translation.
     model = deepcopy(model)
     topology = parse_sklearn_api_model(model)
 
     # Convert the Topology object into a PyTorch model.
-    hb_model = _convert_topology(topology, extra_config=extra_config)
+    hb_model = topology_converter(topology, extra_config=extra_config)
```

Comment:
question: this is now public because the control flow changed?

Reply:
Yes before it was a private method inside convert.py, but now it's a method in its own file. I wanted to call the convert method for each IR the same. Maybe I can have all the IRs subclassing an abstract class with a convert method. I can try to have them using the signature using extra_config or `**kwargs`

Reply:
i think it's ok as-is, i just wanted to make sure i understood why the change. (and maybe something fancier later on)